package.json

```diff
@@ -22,8 +23,12 @@
   },
   "homepage": "https://github.com/follow-the-nft/follow-the-nft-api#readme",
   "dependencies": {
+    "axios": "^0.24.0",
     "dotenv": "^10.0.0",
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "sequelize": "^6.12.0-beta.1",
+    "sequelize-cli": "^6.3.0",
```

Comment:
Remove as `sequelize-cli` should not be a dev dependency but installed locally as a global.
`npm uninstall sequelize-cli && npm install sequelize-cli -g`